omim2obo/parsers/omim_txt_parser.py

```diff
@@ -447,20 +546,27 @@ def get_updated_entries(start_year=2020, start_month=1, end_year=2021, end_month
     return updated_entries
 
 
-def get_hgnc_symbol_id_map(input_path=os.path.join(DATA_DIR, 'hgnc', 'hgnc_complete_set.txt')) -> Dict[str, str]:
-    """Get mapping between HGNC symbols and IDs
-    todo: Ideally download the latest file: http://ftp.ebi.ac.uk/pub/databases/genenames/hgnc/tsv/hgnc_complete_set.txt
-    todo: Address or suppress warning. I dont even need these columns anyway:
-     /Users/joeflack4/projects/omim/omim2obo/main.py:208: DtypeWarning: Columns (32,34,38,40,50) have mixed types.
-     Specify dtype option on import or set low_memory=False.
-     hgnc_symbol_id_map: Dict = get_hgnc_symbol_id_map()
-    """
+# todo: Both HGNC funcs:
+#  1. Address or suppress warning. I dont even need these columns anyway:
+#   ...omim2obo/main.py:208: DtypeWarning: Columns (32,34,38,40,50) have mixed types.
+#   Specify dtype option on import or set low_memory=False.
+#   hgnc_symbol_id_map: Dict = get_hgnc_symbol_id_map()
+#  2. todo: Ideally download latest file: http://ftp.ebi.ac.uk/pub/databases/genenames/hgnc/tsv/hgnc_complete_set.txt
```

Comment:
Does it take more than a `wget` call to download the file? 
NOTE: It looks like on https://www.genenames.org/download/archive/monthly/tsv/ all the files are only available with a date in the file name vs. a "latest" URL also available. I sent a message through the HGNC feedback form to see if there is a more generic "latest" URL available for the hgnc_complete_set file.

I agree, for _right_ now, this is a bit convoluted to download automatically.

Reply:
We should move this from a `todo` to a GitHub issue!

Reply:
HGNC replied back that the "latest" data set is available from: https://storage.googleapis.com/public-download-files/hgnc/tsv/tsv/hgnc_complete_set.txt so a simple `wget` can be used to get the file.

Reply:
Discussed and we will use.